components/ToggleButton.js

```diff
@@ -23,27 +23,24 @@ const breakpoint = 768,
   teal = '#23485f'
 
 const Button = styled.button`
+  @apply px-4 py-2 border-0 text-white;
   appearance: none;
-  color: white;
-  padding: 15px;
-  border: none;
 
   background: ${props => props.isActive ? teal : 'black' };
```

Comment:
I think this would need to be something like

```suggestion
  @apply ${props => props.isActive ? 'bg-teal' : 'bg-black' };
```

RIght?  And I wonder how that would work, like if `@apply` works with one of these interpolated prop functions.  I don't think it would.

On Amp, @isaazgarcia, in cases like this, would you just conditionally toggle classes on and off from `className`?  Like:

```jsx
<Button className={ clsx(
  'px-4 py-2 border-0 text-white appearance-none',
  isActive ? 'bg-teal' : 'bg-black'
)} />
```